assets/js/modules/analytics-4/components/dashboard/AudienceSegmentation/AudienceTiles.stories.js

```diff
@@ -124,7 +124,7 @@ export default {
 
 				registry
 					.dispatch( MODULES_ANALYTICS_4 )
-					.receiveGetAudiences( { audiences: audiencesFixture } );
+					.setAvailableAudiences( { audiences: audiencesFixture } );
```

Comment:
This should provide an array of objects that represent the cached audiences, rather than the underlying fixtures.

I.e. they should have the shape as described in the AC for https://github.com/google/site-kit-wp/issues/8486 (with the exception of the "All Users" `displayName` which should be mapped to "All visitors" as part of this issue).